docs/docs/tutorial.md

```diff
@@ -495,7 +495,6 @@ var CommentForm = React.createClass({
     // TODO: send request to the server
     this.refs.author.getDOMNode().value = '';
     this.refs.text.getDOMNode().value = '';
-    return;
```

Comment:
Our markdown parses uses that stuff between the curly braces to highlight lines - see http://facebook.github.io/react/docs/tutorial.html#adding-new-comments. It looks like this:
![screen shot 2015-02-02 at 12 19 19 pm](https://cloud.githubusercontent.com/assets/8445/6008344/be93a830-aad5-11e4-8ca9-68d573c3c27d.png)

So by removing a line of code, we need to adjust the offsets, otherwise different lines will be highlighted.